apps/server/src/modules/tldraw/service/tldraw.ws.service.spec.ts

```diff
@@ -548,11 +550,39 @@ describe('TldrawWSService', () => {
 			});
 		});
 
+		describe('when deleteUnusedFilesForDocument fails', () => {
+			const setup = async () => {
+				ws = await TestConnection.setupWs(wsUrl);
+				const doc = TldrawWsFactory.createWsSharedDocDo();
+
+				const errorLogSpy = jest.spyOn(logger, 'warning');
+				const storageSpy = jest
+					.spyOn(tldrawFilesStorageAdapterService, 'deleteUnusedFilesForDocument')
+					.mockRejectedValueOnce(new Error('error'));
+
+				return {
+					doc,
+					errorLogSpy,
+					storageSpy,
+				};
+			};
+
+			it('should log error', async () => {
+				const { doc, errorLogSpy } = await setup();
+
+				await service.closeConn(doc, ws);
+				await delay(100);
+
+				expect(errorLogSpy).toHaveBeenCalled();
+				ws.close();
+			});
+		});
+
 		describe('when close connection fails', () => {
 			const setup = async () => {
 				ws = await TestConnection.setupWs(wsUrl);
 
-				const flushDocumentSpy = jest.spyOn(boardRepo, 'flushDocument').mockResolvedValueOnce();
+				const flushDocumentSpy = jest.spyOn(boardRepo, 'compressDocument').mockResolvedValueOnce();
```

Comment:
let boardRepo: DeepMocked<TldrawBoardRepo>;

boardRepo.compressDocument.mockResolvedValueOnce() is the golevel up way for mocking when you have a deep mock. Please avoid jest.spryOn. Only for the service it self you can not avoid it.

Reply:
done, there is probably a lot of stuff to fix in this file because of that change but I'd say not in this PR